src/main/java/org/json/JSONParserConfiguration.java

```diff
@@ -9,6 +9,13 @@ public class JSONParserConfiguration extends ParserConfiguration {
      */
     private boolean overwriteDuplicateKey;
 
+    /**
```

Comment:
Refactoring suggestion: XMLParserConfiguration has an ORIGINAL property. JSONParserConfiguration should probably have one as well. This will make new default instances less expensive.